CONTRIBUTING.md

```diff
@@ -60,21 +60,13 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 
 1. Install [nvm] ([nvm for Windows]), a tool to manage `Node.js` versions.
 2. Restart your terminal so that `nvm` command is recognized.
-3. Using `nvm`, install `Node.js` version `12.22.12`. This is the version used by the [PR CI pipeline] to build `LintDiff`.
-   - `nvm install 12.22.12`
-   - `nvm use 12.22.12`
+3. Using `nvm`, install `Node.js` version `LTS`. This is the version used by the [PR CI pipeline] to build `LintDiff`.
+   - Linux: `nvm install --lts`
+   - Windows: `nvm install lts`
```

Comment:
Recommend installing `lts` which should always be a good default for dev machines, and stays current as versions release over time.